src/dpsvalidationservice/src/test/java/ca/bc/gov/open/pssg/rsbc/dps/dpsvalidationservice/DpsValidationserviceApplicationTests.java

```diff
@@ -7,6 +7,7 @@
 import org.springframework.boot.test.web.client.TestRestTemplate;
 import org.springframework.boot.web.server.LocalServerPort;
 
+import static ca.bc.gov.open.pssg.rsbc.dps.dpsvalidationservice.DpsValidationserviceConstants.VALIDOPEN_DFCMCASE_ERR_RESPONSE;
```

Comment:
Could you please change the import here. See me about how to do this.

Reply:
removed the import